test/files/grd000/generate.py

```diff
@@ -0,0 +1,92 @@
+import ifcopenshell
+import ifcopenshell.template
+import os
+
+rule_code = "grd000"
+
+abs_path = os.path.join(os.getcwd(), "test", "files", rule_code)
+save_ifc_file = lambda file, filename: file.write(os.path.join(abs_path, filename))
+
+f = ifcopenshell.template.create(schema_identifier="IFC4X3_ADD2")
+
+placement = f.createIfcLocalPlacement()
+O = 0., 0., 0.
+X = 1., 0., 0.
+Y = 0., 1., 0.
+Z = 0., 0., 1.
+grid = f.createIfcGrid(
+    GlobalId = ifcopenshell.guid.new(), 
+    ObjectPlacement = placement,
+    UAxes = [f.createIfcGridAxis(
+        AxisTag = "1",
+        AxisCurve = f.createIfcPolyLine([f.createIfcCartesianPoint(O), f.createIfcCartesianPoint(X)]),
+        SameSense = True
+    )],
+    VAxes = [f.createIfcGridAxis(
+        AxisTag = "2",
+        AxisCurve = f.createIfcPolyLine([f.createIfcCartesianPoint(O), f.createIfcCartesianPoint(Y)]),
+        SameSense = True
+    )], 
+    WAxes = [f.createIfcGridAxis(
+        AxisTag = "3",
+        AxisCurve = f.createIfcPolyLine([f.createIfcCartesianPoint(O), f.createIfcCartesianPoint(Z)]),
+        SameSense = True
+    )]
+    )
+
+save_ifc_file(f, f'pass-{rule_code}-not_activated_no_placement.ifc')
+
+
+grid_axis_1 = f.createIfcGridAxis(
+    AxisTag = "Axis 1",
+    AxisCurve = f.createIfcPolyLine([f.createIfcCartesianPoint(O), f.createIfcCartesianPoint(X)]), 
+    SameSense = True
+    )
+
+grid_axis_2 = f.createIfcGridAxis(
+    AxisTag = "Axis 2",
+    AxisCurve = f.createIfcPolyLine([f.createIfcCartesianPoint(O), f.createIfcCartesianPoint(Y)]),
+    SameSense = True
+    )
+
+grid_placement = f.createIfcGridPlacement(
+    PlacementRelTo = placement,
+    )
+
+column = f.createIfcColumn(
+    GlobalId = ifcopenshell.guid.new(),
+    ObjectPlacement = grid_placement,
+)
+
+grid.ObjectPlacement = placement
+
+grid_placement.PlacementLocation = f.createIfcVirtualGridIntersection(
+    [grid_axis_1, grid_axis_2], 
+    (0.0, 0.0, 0.0)
+    )
```

Comment:
This violates the informal propositions here: https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcVirtualGridIntersection.htm

It's also a really creative grid.

I would define:

```
  U axes:
    (0, 0) -> (0, 2)
    (1, 0) -> (1, 2)
    (2, 0) -> (2, 2)
  V axes:
    (0, 0) -> (2, 0)
    (0, 1) -> (2, 1)
    (0, 2) -> (2, 2)
  W:
    <empty>
```

And then you intersect e.g UAxes[1] with VAxes[1], but they need to be instances from the grid itself.

Also you don't actually reference this variable, but redefined sth on L 777

Also the polyline grid axes are best 2d instead of 3d.